common/v2/components/Table.tsx

```diff
@@ -112,14 +113,12 @@ const TableGroupHead = styled(TableRow)`
   cursor: pointer;
 `;
 
-const TableCaret = styled(IconCaret)<{ isFlipped?: boolean }>`
+const TableCaret = styled.img<{ isFlipped?: boolean }>`
   margin-left: 0.5em;
   ${props =>
     props.isFlipped &&
     `
-    svg {
       transform: rotateX(180deg);
-    }
   `};
 `;
```

Comment:
We will most probably need this cheveron in other places.
Let's: 
- delete IconCaret from the project.
- extract this logic into a new component within `v2/components`